server/src/utilities/generateComponent/helpers/__mocks__/reactInstrumentedTs.ts

```diff
@@ -0,0 +1,8 @@
+// This file is being ignored by typescript because it is only used in tests.
```

Comment:
Is there a way to tell the typescript compiler via some glob pattern to ignore certain files in the __mocks__ folder, rather than include `@ts-ignore` within each one?

Reply:
This functionality has been a requested feature in typescript since 2017. https://github.com/Microsoft/TypeScript/issues/19573